arch/risc-v/src/mpfs/mpfs_ddr.c

```diff
@@ -1898,6 +1898,47 @@ static uint8_t mpfs_mtc_test(uint8_t mask, uint64_t start_address,
   return getreg32(MPFS_DDR_CSR_APB_MT_ERROR_STS) & 0x01;
 }
 
```

Comment:
Missing the header:
/****************************************************************************
 * Name: mpfs_mtc_test_all
 *
 * Description:

Reply:
Right, thanks!